src/Run/Setup.cs

```diff
@@ -176,15 +179,39 @@ private void SetRunToolSettingsDefaults()
             {
                 Setting runQuietSetting = new Setting()
                 {
-                    Values = new List<string>() { "True", "False" },
+                    Values = new List<string>() { "true", "false" },
                     ValueType = RunToolSettingValueTypeReservedKeyword,
                     Description = "Run tool specific setting.  Set to True to only display output from the executing command.",
                     DefaultValue = "false"
                 };
 
                 Settings.Add(RunQuietReservedKeyword, runQuietSetting);
             }
-
+            // If WhatIf is already defined in config.json, don't override it
+            if (!Settings.ContainsKey(WhatIfReservedKeyword))
+            {
+                Setting whatIfSetting = new Setting()
+                {
+                    Values = new List<string>() { "true", "false" },
+                    ValueType = RunToolSettingValueTypeReservedKeyword,
+                    Description = "Run tool specific setting.  Set to 'true' to only display commands chosen and not execute them",
+                    DefaultValue = string.Empty
```

Comment:
I would expect that the default should be "true" so that if I just pass `-whatif` it will enable it.